docs/server/v20/server/installation/linux.md

```diff
@@ -10,9 +10,13 @@ If you installed from a pre-built package, the server gets registered as a servi
 sudo systemctl start eventstore
 ```
 
-When you install the EventStoreDB package, the service doesn't start by default. It's done to allow you to change the configuration, located at _/etc/eventstore/eventstore.conf_ and to prevent creating database and index files in the default location.
+When you install the EventStoreDB package, the service doesn't start by default. It's done to allow you to change the configuration, located at `/etc/eventstore/eventstore.conf` and to prevent creating database and index files in the default location.
 
 ::: warning
+Do not start the EventStoreDB service executable (`eventstored`) before starting it as a service. If you do that, all the directories that the server will create will be owned by your user, and the service will not get permissions to write files to those directories until you change the owner to the service user.
```

Comment:
I think this would read better if we said what you should do rather than what you shouldn't? For example 
"Make sure you start eventstore as a service first, This will make sure it gives the service the correct permissions on the log and data directory" @dairyleia thoughts

Reply:
How about "Make sure you start EventStoreDB as a service before starting the service executable (`eventstored`). This will make sure it gives the service the correct permissions on the log and data directory"?